store/tikv/gc_worker.go

```diff
@@ -491,13 +494,11 @@ func (w *GCWorker) loadDurationWithDefault(key string, def time.Duration) (*time
 
 func (w *GCWorker) loadValueFromSysTable(key string) (string, error) {
 	stmt := fmt.Sprintf(`SELECT (variable_value) FROM mysql.tidb WHERE variable_name='%s' FOR UPDATE`, key)
-	restrictExecutor := w.session.(sqlexec.RestrictedSQLExecutor)
-	ctx := w.session.(context.Context)
-	rs, err := restrictExecutor.ExecRestrictedSQL(ctx, stmt)
+	rs, err := w.session.(sqlexec.SQLExecutor).Execute(stmt)
 	if err != nil {
 		return "", errors.Trace(err)
 	}
```

Comment:
I think it's unnecessary.
Because what been executed here is 
`SELECT (variable_value) FROM mysql.tidb WHERE variable_name='%s' FOR UPDATE`. 
**If error check is OK, len(rs)  must be 1.**